src/components/Book.js

```diff
@@ -19,23 +16,29 @@ const Book = ({ details }) => {
         <h3>{title}</h3>
         <p>{author}</p>
         <p>{category}</p>
-      </div>
-      <div className="book-interactions">
-        <button type="button">Comments</button>
-        <button type="button" onClick={() => dispatch(removeBookFromAPI({ itemId }))}>Remove</button>
-        <button type="button">Edit</button>
+        <div className="book-interactions">
+          <button type="button">Comments</button>
+          <button
+            type="button"
+            onClick={() => dispatch(removeBookFromAPI({ itemId }))}
+          >
+            Remove
+          </button>
+          <button type="button">Edit</button>
+        </div>
       </div>
       <div className="book-progress">
         <div className="book-progress-percentage">
-          <p>
-            percentage
-            %
-          </p>
-          <p>Completed</p>
+          <div className="Oval" />
+          <div className="book-progress-percentage-text">
```

Comment:
- [ ] Please kindly ensure that your design matches the original design guidelines (fonts, margins, paddings etc).

### Your Design
![image](https://user-images.githubusercontent.com/43995830/179357544-c7aa9218-2a43-4ee5-bb5d-45b936cb0a14.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/179357529-3ffd7cc4-4f56-4611-a397-5c0f59e056b2.png)

Reply:
@AnselemOdims  Thanks a lot for your review. I have fixed the styles.